document/js-api/index.bs

```diff
@@ -320,12 +330,17 @@ A {{Module}} object represents a single WebAssembly module. Each {{Module}} obje
         1. If [=Type=](|o|) is not Object, throw a {{TypeError}} exception.
         1. Let |v| be [=?=] [$Get$](|o|, |componentName|).
         1. If |externtype| is of the form [=func=] |functype|,
-            1. If [$IsCallable$](|v|) is false, throw a {{LinkError}} exception.
-            1. If |v| has a \[[FunctionAddress]] internal slot, and therefore is an [=Exported Function=],
-                1. Let |funcaddr| be the value of |v|'s \[[FunctionAddress]] internal slot.
-            1. Otherwise,
-                1. [=Create a host function=] from |v| and |functype|, and let |funcaddr| be the result.
-                1. Let |index| be the number of external functions in |imports|. This value |index| is known as the <dfn>index of the host function</dfn> |funcaddr|.
+            1. If [$IsCallable$](|v|) is true
+                1. If |v| has a \[[FunctionAddress]] internal slot, and therefore is an [=Exported Function=],
+                    1. Let |funcaddr| be the value of |v|'s \[[FunctionAddress]] internal slot.
+                1. Otherwise,
+                    1. [=Create a host function=] from |v| and |functype|, and let |funcaddr| be the result.
+            1. If |v| has a \[[wrappedFunction]] insternal slot
```

Comment:
This is not the last condition, so "Otherwise ..." does not seem to be correct.

Reply:
I don't think this needs to be the last condition to use "Otherwise...". It serves the same function as the `else` in an `} else if {`.

As currently written where you do use "Otherwise" for the last step but not this step, it seems that if `[$[IsCallable$](|v|)` is true and `|v|` does not have a `\[[wrapppedFunction]]` internal slot, then a `{{LinkError}}` exception will be thrown, because it seems that the "otherwise" only applies to the previous condition, not both of the previous conditions.

Reply:
My understanding is that using otherwise is exactly like the if then else. There must have been a condition previously. 
However, the text is written in terms of guards:
1. if is callable ...
2. if it has wrappedFunction slot ...

These are actually mutually exclusive: it is not possible for both to be true of the same object.

Reply:
will do